peachjam/js/components/FindDocuments/index.vue

```diff
@@ -308,16 +389,26 @@ export default {
         });
       });
 
+      // Set advanced fields to url
+      Object.keys(this.advancedFields).forEach(key => {
+        const value = this.advancedFields[key];
+        if (!value) return;
+        if (key === 'date' && value.date_from && value.date_to) {
+          params.append('date_from', this.advancedFields.date.date_from);
```

Comment:
I think I tested this and `date=__2022-01-01` and `date=2022-01-01__` are both valid, but double check. ie. we can have open-ended date ranges